sample_data_config/web-api/web-api-data-pipeline.config.yaml

```diff
@@ -0,0 +1,120 @@
+projectName: 'elife-data-pipeline'
+importedTimestampFieldName: 'data_hub_imported_timestamp'
+webApi:
+  #observer api
+  - dataset: '{ENV}'  for observer
+    table: test_published_research_article
+    datataUrl:
+      urlExcludingConfigurableParameters: https://observer.elifesciences.org/report/published-research-article-index?format=json
+  #elifescience people api
+  - dataset: '{ENV}'
+    table: test_people_api
+    datataUrl:
+      urlExcludingConfigurableParameters: https://api.elifesciences.org/people?
+      configurableParameters:
+        pageSize: per-page
+        defaultPageSize: 100
+        page: page
+    response:
+      itemsKeyFromResponseRoot:
+        - items
+  #elifescience profile api
+  - dataset: '{ENV}'
+    table: atest_profile_api
+    datataUrl:
+      urlExcludingConfigurableParameters: https://api.elifesciences.org/profiles?
+      configurableParameters:
+        pageSize: per-page
+        defaultPageSize: 100
+        page: page
+    response:
+      itemsKeyFromResponseRoot:
+        - items
+  #hypoothesis api
+  - dataset: '{ENV}'
+    table: hypothesis_data
+    stateFile:
+      bucketName: 'ci-elife-data-pipeline'
+      objectName: 'airflow-config/generic-web-api/{ENV}-hypothesis-processing-state.json'
+    datataUrl:
+      urlExcludingConfigurableParameters: https://hypothes.is/api/search?group=imRGyeeV&sort=updated&order=asc
+      pagingParameter: search_after
```

Comment:
Is this one used? How come it's outside the `configurableParameters`?

Reply:
option is not used any more, forgot to delete this one in the sample config yaml